Tests/Functional/Filters/Widget/Range/RangeTest.php

```diff
@@ -77,12 +84,38 @@ public function getTestResultsData()
         $out[] = [new Request(['range' => '2;3', 'sort' => '0']), [], true];
 
         // Case #3 invalid range specified.
-        $out[] = [new Request(['range' => '2', 'sort' => '0']), ['1', '2', '3', '4'], true];
+        $out[] = [new Request(['range' => '2', 'sort' => '0']), ['1', '2', '3', '4', '5'], true];
+
+        // Case #4 no range specified.
+        $out[] = [new Request(['sort' => '0']), ['1', '2', '3', '4', '5'], true];
+
+        // Case #5 test with float shouldn't list anything.
+        $out[] = [new Request(['range' => '4.3;50', 'sort' => '0']), [], true];
+
+        // Case #6 test with float should list.
+        $out[] = [new Request(['range' => '4.1;50', 'sort' => '0']), ['5'], true];
 
         return $out;
     }
 
     /**
+     * Check if view data returned is correct.
+     *
+     * @param Request $request Http request.
+     *
+     * @dataProvider getTestResultsData()
```

Comment:
Yeah because testResults in FIlterManagerResultsTest uses the other two. I didn't want to write a separate data provider with the same cases.